companion/src/qcustomplot/GPL.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
yes, it is a third party lib, and it's now in its own directory. (the last version wasn't)